.github/semantic.yml

```diff
@@ -0,0 +1,19 @@
+# Always validate all commits, and ignore the PR title
+commitsOnly: true
```

Comment:
Find a resolution to the bot issue when there are multiple commits?

Reply:
Nop :/